### PR TITLE
Add too dense QR code problem to known issues

### LINKED ISSUE #299: [PE-D][Tester B] QR become unscannable with large fields
Reconstruction steps:
1. edit a person to have a very long name.

Could be mitigated by limiting the maximum length of fields

-------------
Labels: `type.FunctionalityBug` `severity.Low`
original: chiaryan/ped#11

### PR DESCRIPTION
**Pull request by dillontkh**
image

Resolves #299

### CODE REVIEW

**Comment by jjchee77** [comment]
This sounds like it could potentially be marked as a feature flaw during the PE, should we mention something about it in planned enhancement?

File: docs/UserGuide.md [hunk 1]
```diff
@@ -382,6 +382,16 @@ To save a contact to your mobile phone from FitBook, simply scan the QR code nex
 
 1. **When using multiple screens**, if you move the application to a secondary screen, and later switch to using only the primary screen, the GUI will open off-screen. The remedy is to delete the `preferences.json` file created by the application before running the application again.
 
+1. **When trying to scan the QR code of a client with a lot of information stored**, the QR code may become too dense and difficult to scan. The remedy is to reduce the amount of information stored in the client.
+
+> We have verified that the QR code can still be scanned for clients with no more than 100 characters in the following fields:
```
Comment (ruijietay) [inline docs/UserGuide.md:387]:
```suggestion
> We have verified that the QR code can still be scanned for clients with no more than 100 characters in each of the following fields:
```

**Review by ruijietay — CHANGES_REQUESTED** [review]

**Review by RyanNgWH — APPROVED** [review]
lgtm

**Review by ruijietay — APPROVED** [review]
LGTM

**Review by bgopi23 — APPROVED** [review]

**Pull request merged**